.docker/Dockerfile.alpine

```diff
@@ -0,0 +1,91 @@
+ARG ARCH
+ARG NODE_VERSION
+ARG OS
+ARG NODE_RED_VERSION
+
+FROM ${ARCH}/node:${NODE_VERSION}-${OS}
+
+# Basic build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG BUILD_VERSION
+ARG BUILD_REF
+ARG NODE_RED_VERSION
+ARG ARCH
+ARG PYTHON_VERSION
+ARG QEMU_ARCH
+ARG OS
+
+LABEL org.label-schema.build-date=${BUILD_DATE} \
+    org.label-schema.docker.dockerfile=".docker/Dockerfile.alpine" \
+    org.label-schema.license="GNU" \
```

Comment:
What license is this? All of Node-RED project assets are Apache-2.0

(Same comment on all of the Dockerfiles)

Reply:
I'll modify asap!